kable-core/src/commonMain/kotlin/Filter.kt

```diff
@@ -103,7 +103,8 @@ public sealed class Filter {
          */
         public val id: Int,
 
-        public val data: ByteArray,
+        /** Must be non-`null` if [dataMask] is non-`null`. */
```

Comment:
Good point. Gonna check to make sure Topaz's decode covers this.

Reply:
Wasn't covered. Will add.

Reply:
Do we have this for serviceData too?

Reply:
Kable doesn't (yet) support service data filtering. 🙈 
When I do get around to implementing it, then yes, it looks like it would need similar treatment.

Reply:
Created #857 to track.